eclair-core/src/test/scala/fr/acinq/eclair/integration/IntegrationSpec.scala

```diff
@@ -204,8 +204,8 @@ class IntegrationSpec extends TestKit(ActorSystem("test")) with BitcoindService
     connect(nodes("C"), nodes("F3"), 5000000 sat, 0 msat)
     connect(nodes("C"), nodes("F4"), 5000000 sat, 0 msat)
     connect(nodes("C"), nodes("F5"), 5000000 sat, 0 msat)
-    connect(nodes("B"), nodes("G"), 16000000 sat, 0 msat)
-    connect(nodes("G"), nodes("C"), 16000000 sat, 0 msat)
+    connect(nodes("B"), nodes("G"), 46700000 sat, 0 msat)
```

Comment:
I would test the wumbo channel on an Fn channel instead.
Changing the capacity of those is probably messing with the MPP tests (that now may become single-part since there's a lot of capacity). That may be an indication that the MPP/Trampoline tests should be more assertive that multiple parts are actually used...

Reply:
Right! This has been addressed in b70726d, we now create only 1 wumbo channel between C-F1 and we also send a large HTLC (0.4BTC) through it.